services/motion/builtin/move_request.go

```diff
@@ -14,15 +15,19 @@ import (
 	"go.viam.com/rdk/motionplan"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/robot/framesystem"
 	"go.viam.com/rdk/services/motion"
+	"go.viam.com/rdk/services/vision"
 	"go.viam.com/rdk/spatialmath"
 )
 
 // moveRequest is a structure that contains all the information necessary for to make a move call.
 type moveRequest struct {
-	config        *motion.MotionConfiguration
-	planRequest   *motionplan.PlanRequest
-	kinematicBase kinematicbase.KinematicBase
+	config             *motion.MotionConfiguration
+	planRequest        *motionplan.PlanRequest
+	kinematicBase      kinematicbase.KinematicBase
+	cameraVisionPairs  []map[resource.Name]vision.Service
+	frameSystemService framesystem.Service
```

Comment:
Why is this necessary? Can't we use `planRequest.FrameSystem` and `planRequest.StartConfiguration`?

Reply:
These obstacles should be transformables with a parent frame associated, then you should be able to call the transform function to put them into a different desired parent frame

Reply:
We cannot rely on using the transform function to put obstacles into the desired parent frame since the inputs which are passed into the transform function are relative.

Reply:
> We cannot rely on using the transform function to put obstacles into the desired parent frame since the inputs which are passed into the transform function are relative.

That's only the case if cameras, etc in question are not parented to the base and are elsewhere in the World. But in that case, the fs service still doesn't know how to transform them properly.

If the camera is attached to the base, then the camera-base transform should either be static, in which case the transform is easy, or there's something mobile on it (camera on an arm on a base, or camera on a gantry on a base).

Currently we use `referenceframe.StartPositions(baseOnlyFS)` as `planRequest.StartConfiguration`, which would not work for the latter case above; but if we instead initialize StartConfiguration with `fsService.CurrentInputs`, then that should allow for that case.